src/IW/Effects/Download.hs

```diff
@@ -1,27 +1,27 @@
-{- | This module contains the class definitiion of @MonadDownload@ and 
-an instance of @MonadDownload@ for the @App@ monad. Instances of 
+{- | This module contains the class definitiion of @MonadDownload@ and
+an instance of @MonadDownload@ for the @App@ monad. Instances of
 @MonadDownload@ have a @downloadFile@ action for downloading files from a given URL.
 -}
 
 module IW.Effects.Download
        ( MonadDownload (..)
-        
+
        -- * Internals
        , downloadFileImpl
        ) where
 
 import Network.HTTP.Client (Manager, Response (..), httpLbs)
 import Network.HTTP.Types (Status (..))
 
-import IW.App (App, Has, WithError, grab, throwError, notFound)   
-import IW.Core.Url (Url (..)) 
+import IW.App (App, Has, WithError, grab, throwError, notFound)
+import IW.Core.Url (Url (..))
 
 
--- | Describes a monad that can download files from a given URL.
+-- | Describes a monad that can download files from a given @Url@.
 class Monad m => MonadDownload m where
     downloadFile :: Url -> m ByteString
 
-instance MonadDownload App where 
+instance MonadDownload App where
     downloadFile = downloadFileImpl
 
```

Comment:
Oh, I see now that the logging is not added here yet.

Reply:
@vrom911 I could add logging to this PR. But I think it is kinda a lot because I will have to write functions to help convert other library errors to our own, and I have to change our `AppErrorType`:
refer to #110. We're still undecided there I think. Perhaps you could provide your thoughts?

Reply:
@vrom911 I added logging to `MonadDownload1 as well.